atlasdb-client/src/main/java/com/palantir/atlasdb/schema/TargetedSweepSchema.java

```diff
@@ -33,27 +35,48 @@
     INSTANCE;
 
     private static final Namespace NAMESPACE = Namespace.create("sweep");
-    private static final Supplier<Schema> SCHEMA = Suppliers.memoize(() -> generateSchema());
+    private static final Supplier<Schema> SCHEMA = Suppliers.memoize(TargetedSweepSchema::generateSchema);
 
     @SuppressWarnings({"checkstyle:Indentation", "checkstyle:RightCurly"})
     private static Schema generateSchema() {
-        Schema schema = new Schema("TargetedSweep",
+        Schema schema = newSchemaObject();
+        addSweepStateTables(schema);
+        addTableIdentifierTables(schema);
+
+        schema.validate();
+        return schema;
+    }
+
+    /**
+     * This is visible for the ETE tests, which must not truncate the table identifier tables or risk cache
```

Comment:
Maybe worth clarifying that the table mappings can be persistent without any issues?